ZenPacks/zenoss/OpenStackInfrastructure/datasources/NeutronAgentStatusDataSource.py

```diff
@@ -196,7 +196,7 @@ def onSuccess(self, result, config):
             'summary': 'Neutron Agent Status Collector: successful collection',
             'severity': ZenEventClasses.Clear,
             'eventKey': 'openStackNeutronAgentCollectionError',
-            'eventClassKey': 'openStackFailure',
+            'eventClassKey': 'openstack-Failure',
```

Comment:
This is probably a good time to revisit this while we're making changes.   I think that the best practice is to have a single eventClassKey for each of these events, and no eventKey. 

Right now we appear to have eventKeys used for each kind of event, but then separate class keys for failure and success- which I don't think there's any reason for other than that I probably didn't know what I was doing when I set it up originally.

It's also possible that I still don't.

Reply:
I updated the eventClassKeys but I left in the eventKeys since I believe they are needed for event auto clearing